openstack/networking/v2/extensions/agents/results.go

```diff
@@ -126,3 +127,19 @@ func ExtractAgents(r pagination.Page) ([]Agent, error) {
 	err := (r.(AgentPage)).ExtractInto(&s)
 	return s.Agents, err
 }
+
+// GetDhcpNetworksResult is the response from a Get operation.
+// Call its Extract method to interpret it as networks.
+type GetDhcpNetworksResult struct {
```

Comment:
Should this be `ListDHCPNetworksResult` instead of `Get...`?

Reply:
Thanks, will adapt it.